### PR TITLE
[ENG-1138] Mobile view navigation bar

### PR DESCRIPTION
**Pull request by baylee-d**
- Ticket: [ENG-1138]
- Feature flag: n/a

## Purpose

To add a navigation bar on the registration submission page for mobile view

## Summary of Changes

- Add navigation menu for mobile
- Add error message below navigation bar on mobile

## Side Effects

None

## QA Notes

None

[ENG-1138]: https://openscience.atlassian.net/browse/ENG-1138

### CODE REVIEW

File: lib/registries/addon/drafts/draft/styles.scss [hunk 1]
```diff
@@ -80,3 +79,18 @@
         box-shadow: none;
     }
 }
+
+.SideBarToggle {
+    width: 100%;
+}
+
+.NonActive {
```
Comment (aaxelb) [inline lib/registries/addon/drafts/draft/styles.scss]: is this class used?

File: lib/registries/addon/drafts/draft/-components/register/styles.scss [hunk 2]
```diff
@@ -12,3 +12,16 @@
...
+.mobileReviewButtonItem {
+    height: 100%;
+}
+
+.register-button.mobileReviewButtonItem {
+    border-radius: 0;
+    padding: 10px;
+}
+
+.reviewTooltip {
```
Comment (aaxelb) [inline lib/registries/addon/drafts/draft/-components/register/styles.scss]: is this class used?

File: lib/registries/addon/drafts/draft/template.hbs [hunk 3]
```diff
@@ -48,6 +100,17 @@
                 </layout.leftNav>
                 <layout.main local-class='Main'>
                     {{#if this.inReview}}
+                        {{#unless draftManager.registrationResponsesIsValid}}
+                            {{#if this.showMobileView}}
```
Comment (jamescdavis) [inline lib/registries/addon/drafts/draft/template.hbs]: I don't love this construct, but it's even grosser combined. We should consider creating something like `draftManager.hasInvalidResponses` that would make these conditions read better:
```hbs
{{#if (and draftManager.hasInvalidResponses this.showMobileView)}}
```
and for the register button:
```hbs
{{#if (and draftManager.hasInvalidResponses (not @showMobileView))}}
```
Or possibly just created computed for each:
```hbs
{{#if this.showInvalidResponsesMessageInMain}}
```
```hbs
{{#if this.showInvalidResponsesMessageInRight}}
```

Comment (jamescdavis) [reply lib/registries/addon/drafts/draft/template.hbs]: I suppose the latter is local to the `Register` component so maybe should just be:
```hbs
{{#if this.showInvalidResponsesMessage}}
```

**Pull request merged**